packages/client/src/utils.test.ts

```diff
@@ -32,3 +32,54 @@ describe('decodeToken', () => {
     expect(() => decodeToken(JWT)).toThrowError(StructError);
   });
 });
+
+describe('generateCallbackUri', () => {
+  test('generate callback url with redirectUri', () => {
+    const redirectUri = 'http://localhost:3000';
```

Comment:
normally we could name using all uppercase to indicate it is a static constant. i.e.  `REDIRECT_URI`